### PR TITLE
Added recipe for cov-core

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Seems like a weird Appveyor bug on `python 34`. Close/reopen...

File: recipes/cov-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    # - coverage >=3.6
```
Comment (jakirkham) [inline recipes/cov-core/meta.yaml]: Looks like we can drop this now. 😄

**Comment by jakirkham** [comment]
One comment then LGTM.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**